tools/v8_gypfiles/v8.gyp

```diff
@@ -847,7 +847,7 @@
           'conditions': [
             ['v8_enable_webassembly==1', {
               'conditions': [
-                ['OS=="mac" or (_toolset=="host" and host_arch=="x64" and OS=="linux")', {
+                ['OS=="mac" or OS=="ios" or (_toolset=="host" and host_arch=="x64" and (host_os=="linux" or host_os=="mac"))', {
```

Comment:
Minor thing but you can write this as:
```suggestion
                ['OS=="mac" or OS=="ios" or '
                 '(_toolset=="host" and host_arch=="x64" and host_os in "linux mac")', {
```
Now you also know how to wrap at 80 columns. :-)

Reply:
You mean a row have to be less than 80 columns? But there is many line exceeds 80 columns, include line 851 you demonstrated. :-)

Reply:
Sure, but I'd like to see it get better over time, not worse. This line in particular is pretty egregious.

Reply:
ok